### PR TITLE
Add some Shimmmer to the wiki

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added an announcement bar for Shimmer.
wiki_announcement_bar

## How the change has been tested

Tested locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Review by jlvandenhout — CHANGES_REQUESTED** [review]
Also maybe the background color can be a more friendly notification color instead of the bright green.

**Comment by Dr-Electron** [comment]
> Also maybe the background color can be a more friendly notification color instead of the bright green.

But it is the Shimmer color 😿

**Review by jlvandenhout — APPROVED** [review]

**Comment by Dr-Electron** [comment]
> That's good enough a reason for me :D

That was easy 😅

**Review by Critical94 — APPROVED** [review]
I like the color :D

File: docusaurus.config.js [hunk 1]
```diff
@@ -21,6 +21,14 @@ module.exports = {
     },
   ],
   themeConfig: {
+    announcementBar: {
+      id: 'shimmer',
+      content:
+        'Interested in the upcoming incentivized staging network and staking IOTA tokens? Read more about it <a target="_blank" href="https://shimmer.network">here</a>.',
```
Comment (Phyloiota) [inline docusaurus.config.js:27]:
```suggestion
        'Interested in the upcoming incentivized staging network Shimmer, and in staking your IOTA tokens? Read more about it <a target="_blank" href="https://shimmer.network">here</a>.',
```

**Review by Phyloiota — APPROVED** [review]

**Pull request merged**